src/components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useState, useMemo, useEffect } from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+import {
+  postTodo,
+  deleteTodo,
+  getTodo,
+  patchTodos,
+} from '../../api/todos/todosAPI';
+
+import { TodoList } from '../TodoList';
+import './TodoApp.scss';
+
+type Props = {
+  todos: Todo[],
+  user: User | undefined,
+  removeUser: () => void,
+  handlerSingOut: () => void,
+};
+
+export const TodoApp: React.FC<Props> = ({
+  todos,
+  user,
+  removeUser,
+  handlerSingOut,
+}) => {
+  const [visibleTodo, setTodo] = useState<Todo[]>(todos);
```

Comment:
```suggestion
  const [visibleTodos, setVisibleTodos] = useState<Todo[]>(todos);
```
в useState хуці потрібно дотримуватись таких неймінгів